src/components/Buttons/Button.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  onClick: () => void;
+  type?: 'button' | 'submit' | 'reset';
+  className?: string;
+  small?: boolean;
+  large?: boolean;
+  outline?: boolean;
+};
+
+// TODO: share style with ButtonLink
+const button =
+  'inline-flex items-center justify-center rounded-full py-2 px-4 drop-shadow border-2 border-emerald-300';
+const defaultButtonStyle =
+  'bg-emerald-300 text-slate-600 transition-colors hover:bg-slate-800 hover:text-emerald-300 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2';
+const outlineButtonStyle =
+  'bg-white font-bold text-slate-600 transition-colors hover:bg-emerald-300 hover:text-slate-600 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2';
+
+export const Button: React.FC<Props> = ({
+  onClick,
+  type = 'button',
+  className = '',
```

Comment:
```suggestion
  className,
```

Braucht keine Default. `classNames` regelt das undefined.